.github/workflows/manual-translation-update.yml

```diff
@@ -12,33 +11,40 @@ on:
 jobs:
   update-translations:
     runs-on: ubuntu-latest
-
+    name: "Download latest translations from Phrase.com and make a PR."
     steps:
       - name: Check out repo
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29
         with:
           fetch-depth: 0
-          token: ${{ secrets.RELEASE_ACCESS_KEY }}
+          token: ${{ secrets.RELEASE_ACCESS_TOKEN }}
 
       - name: Install Phrase CLI
         uses: phrase/setup-cli@d1c415d0ff01efc1bb21287d60a547669d9331c5
         with:
-          version: '2.19.0'
+          version: '2.29.0'
 
-      - name: Authenticate Phrase CLI
-        run: phrase auth login --access-token ${{ secrets.PHRASE_ACCESS_KEY }}
+      - name: Prepend Phrase configuration
+        run: |
+          echo "phrase:" > .phrase.yml
+          echo "  access_token: ${PHRASE_ACCESS_KEY}" >> .phrase.yml
+          echo "  project_id: ${PHRASE_PROJECT_ID}" >> .phrase.yml
+          cat phrase_config.yml >> .phrase.yml
+        env:
+          PHRASE_ACCESS_KEY: ${{ secrets.PHRASE_ACCESS_KEY }}
+          PHRASE_PROJECT_ID: ${{ secrets.PHRASE_PROJECT_ID }}
 
       - name: Download Translations from Phrase
-        run: phrase pull --project-id ${{ secrets.PHRASE_PROJECT_ID }} --branch ${{ github.event.inputs.branch }}
+        run: phrase pull
 
       - name: Commit and Push Changes
         run: |
           git config --global user.email "action@github.com"
```

Comment:
We dont need the --global here or with user.name `git config user.email action@github.com` will do - also no need for the quote marks (although is probably fine with them).